chain-crypto/src/algorithms/vrf/vrf.rs

```diff
@@ -208,7 +189,7 @@ impl ProvenOutputSeed {
             return None;
         }
         let u = GroupElement::from_bytes(&bytes[0..GroupElement::BYTES_LEN])?;
-        let proof = dleq::Proof::from_bytes(&bytes[GroupElement::BYTES_LEN..])?;
+        let proof = dleq::Zkp::from_slice(&bytes[GroupElement::BYTES_LEN..])?;
```

Comment:
Bikeshedding: I think `from_bytes` is better as it's clear we mean to convert from a byte array in this serialization API, while `from_slice` typically is used for slices of elements of some non-primary type.
We use `from_bytes` everywhere, let's keep to the convention.